src/hooks/my/useItemsPerPage.jsx

```diff
@@ -1,27 +1,36 @@
 import { useState, useEffect } from 'react';
 
-function useItemsPerPage() {
-    const [itemsPerPage, setItemsPerPage] = useState(16);
+// 화면 크기를 나타내는 상수 객체
+const SIZES = {
+    mobile: 768,
+    tablet: 1280,
+};
+
+// 화면 크기에 따라 보여줄 아이템 개수를 동적으로 설정
+function useItemsPerPage(items = { mobile: 6, tablet: 8, desktop: 16 }) {
+    const [itemsPerPage, setItemsPerPage] = useState(items.desktop);
 
     useEffect(() => {
+        // 화면 크기에 따라 아이템 개수를 업데이트하는 함수
         const updateItemsPerPage = () => {
```

Comment:
```suggestion
    useEffect(() => {
        const updateItemsPerPage = () => {
            setItemsPerPage(getItemsPerPage());
        };

        window.addEventListener('resize', updateItemsPerPage); 
```
getItemsPerPage를 useEffect 밖으로 빼면 더 깔끔할  것 같아요!